guides/doc-Upgrading_and_Updating/topics/introduction_updating_satellite.adoc

```diff
@@ -1,6 +1,6 @@
 [[introduction_updating_satellite]]
```

Comment:
The title for this is rendering as: 

Updating Satellite Server, Capsule Server, and Content Hosts